src/runtime/actor/handle.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use thiserror::Error;
+use tokio::sync::mpsc::UnboundedSender;
+
+use super::{
+    envelope::{DynEvent, Envelope},
+    Actor,
+};
+use crate::runtime::{registry::ScopeId, scope::ScopeView};
+
+/// Error sending a message to an actor
+#[derive(Error, Debug)]
+#[error("Error sending message to actor: {0}")]
+pub struct SendError(String);
+
+#[allow(missing_docs)]
+impl SendError {
+    pub fn new<S: Into<String>>(msg: S) -> Self {
+        Self(msg.into())
+    }
+}
+
+impl<S: Into<String>> From<S> for SendError {
+    fn from(msg: S) -> Self {
+        Self::new(msg)
+    }
+}
+
+/// An actor handle, used to send events
+#[derive(Debug)]
+pub struct Addr<A: Actor> {
+    pub(crate) scope: ScopeView,
+    pub(crate) sender: UnboundedSender<Envelope<A>>,
+}
+
+impl<A: Actor> Addr<A> {
+    pub(crate) fn new(scope: ScopeView, sender: UnboundedSender<Envelope<A>>) -> Self {
+        Self { scope, sender }
+    }
+
+    /// Shut down the actor with this handle. Use with care!
```

Comment:
In this and the API docs below you use an imperative style except for the `is_closed` method. There is a guideline on this one can find online, and despite not being followed rigorously sometimes, especially when it comes to trait fns like for `Read` and `Write`, in Bee we decided to consistently use 3rd person singular in the docs according to that aforementioned API guideline. If you find those nit-picky things annoying then blame this on my german-ess please :trollface: Also please use proper punctuation. :walking:

Reply:
no